requirements-dev.txt

```
@@ -1,13 +1,13 @@
 black==23.7.0
 build==0.10.0
-flake8==6.0.0
+flake8==6.1.0
 invoke==2.2.0
 isort==5.12.0
 jsondiff==2.0.0
 mypy==1.4.1
 pre-commit==3.3.3
-pylint==2.17.4
+pylint==2.17.5
 pytest==7.4.0
 pytest-cov==4.1.0
-ruff==0.0.278
+ruff==0.0.280
```

```suggestion
ruff==0.0.281
```
It is a bit strange that the precommit -config uses a version that is newer than that in the requirements.